HyperVExtension/src/DevSetupAgent/RequestManager.cs

```diff
@@ -81,24 +96,28 @@ public void ProcessRequestMessage(IRequestMessage message, CancellationToken sto
 
     private void ProcessRequestQueue(CancellationToken stoppingToken)
     {
-        while (!stoppingToken.IsCancellationRequested)
+        while (true)

```

Comment:
Why the switch to `while(true)`?

Reply:
I moved check IsCancellationRequested under the queue lock to always set state variables correctly and have exit out of the loop in one place only.